test/org/zaproxy/zap/extension/pscanrulesAlpha/XBackendServerInformationLeakUnitTest.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2018 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.pscanrulesAlpha;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+import org.apache.commons.httpclient.URI;
+import org.apache.commons.httpclient.URIException;
+import org.junit.Before;
+import org.junit.Test;
+import org.parosproxy.paros.network.HttpMessage;
+import org.parosproxy.paros.network.HttpRequestHeader;
+import org.zaproxy.zap.extension.pscan.PluginPassiveScanner;
+
+public class XBackendServerInformationLeakUnitTest extends PassiveScannerTest {
+
+	private static final String XBS_HEADER = "X-Backend-Server";
+	private static final String HEADER_VALUE = "developer1.webapp.scl3.mozilla.com";
+	private HttpMessage msg;
+
+	@Before
+	public void before() throws URIException {
+		HttpRequestHeader requestHeader = new HttpRequestHeader();
+		requestHeader.setURI(new URI("http://example.com", false));
+
+		msg = new HttpMessage();
+		msg.setRequestHeader(requestHeader);
+	}
+
+	@Override
+	protected PluginPassiveScanner createScanner() {
+		return new XBackendServerInformationLeak();
+	}
+
+	@Test
+	public void shouldNotRaiseAlertIfResponseHasNoRelevantContent() {
+		// Given
+		msg.setResponseBody("Some text <h1>Some Title Element</h1>");
```

Comment:
I'd not add anything since the body does not matter for the tests. `// Given - default msg` like in other tests?